src/client/systems/ClientGameController.cpp

```diff
@@ -0,0 +1,168 @@
+#include <thread>
```

Comment:
did you use `git mv` to move these files? it destroys the git history for the file otherwise. pretty annoying